test/sub-menu.html

```diff
@@ -189,6 +331,17 @@
           await nextRender(subMenu);
           expect(buttons[0].hasAttribute('expanded')).to.be.false;
         });
+
+        it('should remove expanded attribute and restore focus when sub-menu closed on Esc', async() => {
+          arrowDown(buttons[0]);
+          await nextRender(subMenu);
+          expect(buttons[0].hasAttribute('expanded')).to.be.true;
+
+          MockInteractions.pressAndReleaseKeyOn(subMenu.$.overlay, 27, [], 'Escape');
```

Comment:
I guess this could use the `esc` helper function? not blocking

Reply:
Thanks, done.